build.gradle

```diff
@@ -0,0 +1,49 @@
+buildscript {
+    ext.kotlin_version = '1.1.51'
+
+    repositories {
+        mavenCentral()
+        maven { url 'https://dl.bintray.com/kotlin/kotlin-dev' }
+        maven { url 'http://dl.bintray.com/jetbrains/intellij-plugin-service' }
+    }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+plugins {
+    id "org.jetbrains.intellij" version "0.2.5"
+}
+
+apply plugin: 'java'
+apply plugin: 'kotlin'
+apply plugin: 'org.jetbrains.intellij'
+
+sourceSets {
+    main.java.srcDirs += 'src'
+    test.java.srcDirs += 'test'
```

Comment:
adding this line helped me solve the compile errors
```gradle
test.compileClasspath += files(compileKotlin.destinationDir)
```
though I'm pretty sure it shouldn't be necessary.

Reply:
Hi @ajermakovics 

The problem is IntelliJ plugin is outdated. So I upgrade it from `0.2.5` to `0.2.17` and everything work fine.

Reply:
Indeed, works great